### PR TITLE
request retries are deactivated by default on mutations

### LINKED ISSUE #2154: Add an option to deactivate subgraph request retries for mutations
mutations may not be idempotent, so if we get an error when sending one, we might choose to return an error instead of retrying

### PR DESCRIPTION
**Pull request by Geal**
Fix #2154
Follow up on #2006

### CODE REVIEW

**Review by bnjjj — APPROVED** [review]

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -205,13 +207,14 @@ traffic_shaping:
       min_per_sec: 10
       ttl: 10s
       retry_percent: 0.2
+      retryable_mutations: false
```
Comment (BrynCooke) [inline NEXT_CHANGELOG.md:210]: Suggest `retryable_mutations` -> `retry_mutations`.

**Review by garypen — APPROVED** [review]
Approving but also think `retry_mutations` is a better name.

File: NEXT_CHANGELOG.md (hunk 1 above)
Comment (Geal) [reply NEXT_CHANGELOG.md]: fixed in c47aafd

**Comment by Geal** [comment]
> Approving but also think retry_mutations is a better name.
same

**Pull request merged**